bin/sail

```diff
@@ -210,6 +210,18 @@ if [ $# -gt 0 ]; then
             sail_is_not_running
         fi
 
+    # Initiate a PostgreSQL CLI terminal session within the "psql" container...
+    elif [ "$1" == "psql" ]; then
```

Comment:
I'm curious why you called this "psql" even though the container is "pgsql"?

Reply:
because the PostgreSQL client command is `psql` (but I'm curious why this is, too 😇 ):
https://www.postgresql.org/docs/13/app-psql.html